src/frontend/src/binder/expr/function.rs

```diff
@@ -154,12 +154,19 @@ impl Binder {
             .into());
         }
```

Comment:
```suggestion
        } else {
        
        }
```
we need also check the Ordered-Set Agg Kinds followed with a within group clause